samples/browser/src/main/java/org/mozilla/samples/browser/DefaultComponents.kt

```diff
@@ -62,9 +61,9 @@ open class DefaultComponents(private val applicationContext: Context) {
             }
 
             sessionStorage.autoSave(this)
-                .periodicallyInForeground(interval = 30, unit = TimeUnit.SECONDS)
+                // .periodicallyInForeground(interval = 30, unit = TimeUnit.SECONDS)
                 .whenGoingToBackground()
-                .whenSessionsChange()
+                // .whenSessionsChange()
```

Comment:
I guess we do not want to land this part? :)

Reply:
Nope. :) That's just to prevent the app from freezing when testing. We can schedule a storage/autosave fix for the next sprint....